src/main/java/com/database/data/DBCloseConnect.java

```diff
@@ -0,0 +1,29 @@
+package com.database.data;
+
+import java.sql.Connection;
+import java.sql.Statement;
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+import java.sql.ResultSet;
+
+public class DBCloseConnect {
+
+    public static void close(Connection connection, PreparedStatement preparedStatement) throws SQLException {
+        preparedStatement.executeUpdate();
+        connection.commit();
+        preparedStatement.close();
+        connection.close();
+    }
+
+    public static void close(ResultSet resultSet, Connection connection, PreparedStatement preparedStatement) throws SQLException {
```

Comment:
Why did you need this method and the next one, if PreparedStatement is extended from Statement?
![image](https://user-images.githubusercontent.com/20267148/55959487-90b0a180-5c73-11e9-92c8-ba039e4b580b.png)